tests/test_datasets/test_dataset_functions.py

```diff
@@ -317,12 +319,19 @@ def test_deletion_of_cache_dir_faulty_download(self, patch):
         self.assertEqual(len(os.listdir(datasets_cache_dir)), 0)
 
     def test_publish_dataset(self):
+
         openml.datasets.get_dataset(3)
         file_path = os.path.join(openml.config.get_cache_directory(),
                                  "datasets", "3", "dataset.arff")
         dataset = OpenMLDataset(
-            "anneal", "test", "ARFF",
-            version=1, licence="public", default_target_attribute="class", data_file=file_path)
+            "anneal",
+            "test",
+            data_format="ARFF",
```

Comment:
uhm, it should not be `"arff"` in lower case even this is converted after.

Reply:
you mean it should be "arff".

Reply:
Yep